### PR TITLE
[DC-2411] Update all bq.build_and_copy_contents(...) calls in curation

### PR DESCRIPTION
**Pull request by lledezma**

### CODE REVIEW

File: data_steward/gcloud/bq/__init__.py [hunk 1]
```diff
@@ -405,3 +406,40 @@ def update_labels_and_tags(self,
             return {**existing_labels_or_tags, **updates}
 
         return {**existing_labels_or_tags, **updates}
+
+    def build_and_copy_contents(self, src_dataset: str, dest_dataset: str):
```
Comment (ksdkalluri) [inline data_steward/gcloud/bq/__init__.py:410]: @lledezma this function in util/bq is updated recently, it would be great to update this as well.

**Review by ksdkalluri — CHANGES_REQUESTED** [review]

File: data_steward/gcloud/bq/__init__.py (hunk 1 above)
Comment (lledezma) [reply data_steward/gcloud/bq/__init__.py:410]: In this ticket or a separate ticket?

Comment (ksdkalluri) [reply data_steward/gcloud/bq/__init__.py:410]: @lledezma tools/create_tier still uses bq.build_and_copy_contents() please upthat that as well.

**Review by ksdkalluri — CHANGES_REQUESTED** [review]

**Review by ksdkalluri — APPROVED** [review]
Thanks for the changes @lledezma!

**Pull request merged**